images/docker-compose.yml

```diff
@@ -2,7 +2,7 @@ version: "3"
 
 services:
   stellar-core-1:
-    image: kinecosystem/stellar-core:kinecosystem-v2.0.0-stellar-v9.2.0
+    image: kinecosystem/stellar-core:latest
```

Comment:
It's better to hardcode the version. This is the only way to be 100% sure the build is stable. Future updates might break the code.